EIPS/eip-3534.md

```diff
@@ -0,0 +1,318 @@
+---
+eip: 3534
+title: Restricted Chain Context Type Transactions
+author: Isaac Ardis (@whilei)
+discussions-to: https://ethereum-magicians.org/t/eip-3534-restricted-chain-context-transaction-type/6112
+status: Draft
+type: Standards Track
+category: Core
+created: 2021-04-20
+requires: 2718, 2930
```

Comment:
Since EIP-1559 will be likely be included in the next hard fork, you may want to consider having this depend on EIP-1559 rather than EIP-2930 and be formatted in a way compatible with 1559 transactions.

Reply:
Well noted. I'll leave it as-is for now, but will keep this in mind.